src/main/java/io/strimzi/kafka/bridge/amqp/AmqpConfig.java

```diff
@@ -0,0 +1,160 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.strimzi.kafka.bridge.amqp;
+
+import java.util.Map;
+
+/**
+ * AMQP related configuration
+ */
+public class AmqpConfig {
+
+    private static final String AMQP_MODE = "AMQP_MODE";
+    private static final String AMQP_HOST = "AMQP_HOST";
+    private static final String AMQP_PORT = "AMQP_PORT";
+    private static final String AMQP_FLOW_CREDIT = "AMQP_FLOW_CREDIT";
+    private static final String AMQP_MESSAGE_CONVERTER = "AMQP_MESSAGE_CONVERTER";
+    private static final String AMQP_CERT_DIR = "AMQP_CERT_DIR";
+
+    private static final String DEFAULT_AMQP_MODE = "SERVER";
+    private static final String DEFAULT_HOST = "0.0.0.0";
+    private static final int DEFAULT_PORT = 5672;
+    private static final int DEFAULT_FLOW_CREDIT = 1024;
+    private static final String DEFAULT_MESSAGE_CONVERTER = "io.strimzi.kafka.bridge.amqp.converter.AmqpDefaultMessageConverter";
+    private static final String DEFAULT_CERT_DIR = null;
+
+    private AmqpMode mode;
+    private int flowCredit;
+    private String host;
+    private int port;
+    private String messageConverter;
+    private String certDir;
+
+    /**
+     * Constructor
+     *
+     * @param mode the AMQP bridge working mode (client or server)
+     * @param flowCredit the AMQP receiver flow credit
+     * @param host the host for AMQP client (to connect) or server (to bind)
+     * @param port the port for AMQP client (to connect) or server (to bind)
+     * @param messageConverter the AMQP message converter
+     * @param certDir the directory with the TLS certificates files
+     */
+    public AmqpConfig(AmqpMode mode, int flowCredit, String host, int port, String messageConverter, String certDir) {
+        this.mode = mode;
+        this.flowCredit = flowCredit;
+        this.host = host;
+        this.port = port;
+        this.messageConverter = messageConverter;
+        this.certDir = certDir;
+    }
+
+    /**
+     * @return the AMQP bridge working mode (client or server)
+     */
+    public AmqpMode getMode() {
+        return this.mode;
+    }
+
+    /**
+     * @return the AMQP receiver flow credit
+     */
+    public int getFlowCredit() {
+        return this.flowCredit;
+    }
+
+    /**
+     * @return the host for AMQP client (to connect) or server (to bind)
+     */
+    public String getHost() {
+        return this.host;
+    }
+
+    /**
+     * @return the port for AMQP client (to connect) or server (to bind)
+     */
+    public int getPort() {
+        return this.port;
+    }
+
+    /**
+     * @return the AMQP message converter
+     */
+    public String getMessageConverter() {
+        return this.messageConverter;
+    }
+
+    /**
+     * Set the AMQP message converter
+     *
+     * @param messageConverter  AMQP message converter
+     * @return  this instance for setter chaining
```

Comment:
two spaces after messageConverter and @return ( Doesn't matter, I am just showing off my eagle eyes :p )